opa-iptables/pkg/iptables/rule.go

```diff
@@ -0,0 +1,219 @@
+package iptables
+
+import (
+	"strings"
+	"fmt"
+	"bytes"
+)
+
+// Rule reperesents an IPTable rule
+type Rule struct {
```

Comment:
Should we add "omitempty" to add of these in case of (Un)Marshal? I am asking, not sure of your further plans

Reply:
If user doesn't provides value, In that case default value will be taken. for string it's default value is "" and in case of slice is []string{} is return during Unmarshal. Because we are checking
each fields of Rule struct for emptiness in addParam() function we don't need "omitempty" flag.